website/docs/getting_started.html.markdown

```diff
@@ -0,0 +1,224 @@
+---
+layout: "google"
+page_title: "Getting Started with the Google provider"
+sidebar_current: "docs-google-provider-getting-started"
+description: |-
+  Getting started with the Google Cloud Platform provider
+---
+
+# Getting Started with the Google Provider
+
+## Before you begin
+
+* You'll want to have created an account and project in the [Google Cloud Console](https://console.cloud.google.com/)
+and set up billing on that project. Any examples in this guide will be part of
+the [GCP "always free" tier](https://cloud.google.com/free/).
+* [Install Terraform](https://www.terraform.io/intro/getting-started/install.html)
+and read the Terraform getting started guide that follows. This guide will
+assume basic proficiency with Terraform - it is an introduction to the Google
+provider.
+
+## Configuring the Provider
+
+To configure the provider, first create a Terraform config file. Inside, you'll
```

Comment:
Since this is in a section called "configuring the provider", we can probably remove "to configure the provider"